FogCreek/Glitch-Community
# Cuddly dormouse: componentize collection lists

## Links
* Remix link: https://cuddly-dormouse.glitch.me
* Manuscript link: https://glitch.manuscript.com/f/cases/3328387/

## GIF/Screenshots
Team with empty collections https://cuddly-dormouse.glitch.me/@community
![Screen Shot 2019-04-29 at 5 44 39 PM](https://user-images.githubusercontent.com/938722/56929231-12436480-6aa7-11e9-951a-11017d22dbd4.png)

Team has collections https://cuddly-dormouse.glitch.me/@glitch
![Screen Shot 2019-04-29 at 5 45 07 PM](https://user-images.githubusercontent.com/938722/56929232-12436480-6aa7-11e9-9f9b-27eee04a095c.png)

More from collection curator https://cuddly-dormouse.glitch.me/@glitch/glitch-this-week-april-24-2019
![Screen Shot 2019-04-29 at 5 45 24 PM](https://user-images.githubusercontent.com/938722/56929233-12436480-6aa7-11e9-8dc4-b6715f7d7d30.png)

Included in collections https://cuddly-dormouse.glitch.me/~csv-name-randomizer
![Screen Shot 2019-04-29 at 5 46 39 PM](https://user-images.githubusercontent.com/938722/56929234-12436480-6aa7-11e9-8645-cd5f8d7b40ec.png)

## Changes:
* componentize CollectionList
* componentize MoreCollections **(now using CollectionItemSmall)**
* inline IncludedInCollections into project page
* add `showCurator` to CollectionItemSmall
* delete unused CSS

## How To Test:
* visit https://cuddly-dormouse.glitch.me and log in
* verify collection lists render correctly on the above pages
